### PR TITLE
align ports and basic auth

### PR DESCRIPTION
**Pull request by rhiGriff**
Align default port numbers for all apps and default basic auth. Port numbers for all apps can be found here: https://digitaleq.atlassian.net/wiki/spaces/RASB/pages/94863418/Developer+Environment+Port+Mappings

### CODE REVIEW

File: run.py [hunk 1]
```diff
@@ -13,6 +13,6 @@
 logger = wrap_logger(logging.getLogger(__name__))
 
 if __name__ == '__main__':
-    port = os.getenv('FS_DEV_PORT', 8080)
+    port = os.getenv('FS_DEV_PORT', 8082)
```
Comment (benjefferies) [inline run.py]: Dockerfile needs updating too

Comment (rhiGriff) [reply run.py]: This shouldn't affect docker

Comment (benjefferies) [reply run.py]: The expose port should reflect the port the app is listening on https://github.com/ONSdigital/ras-frontstage/blob/master/Dockerfile#L6. You'll also need to update the docker compose files https://github.com/ONSdigital/ras-rm-docker-dev/blob/master/.env

Comment (madeye-matt) [reply run.py]: @benjefferies I've spoken to Rhi - I am going to take a look at ras-rm-docker-dev repo - see if we can align the ports there with the ports in Rhi's list (from link at top)

**Comment by rhiGriff** [comment]
[Align all application port numbers to run locally](https://trello.com/c/DNzTTGD7/31-align-all-application-port-numbers-to-run-locally)

File: run.py (hunk 1 above)
Comment (jcox-dev) [inline run.py]: This is doing something different to the other repos and not using the config for the port number. Is that ok?

Comment (andrewmil) [reply run.py]: This is something left over from way back when frontstage was started, would be good to bring it in line with the other repos

Comment (jcox-dev) [reply run.py]: Attached a simple patch for [using config](https://github.com/ONSdigital/ras-frontstage/files/1606695/use_config.txt).

**Comment by jcox-dev** [comment]
@gemmai95 true that. I should have just dismissed the "requested changes" 👍

**Pull request merged**